src/components/mainMenu.vue

```diff
@@ -83,9 +83,17 @@ export default defineComponent({
         });
       }
       if (this.launchOnStart) {
-        await this.$autoLauncher.enable();
+        try {
+          await this.$autoLauncher.enable();
+        } catch (error) {
+          this.store.setError({ title: 'errorPage.enableAutoLauncherFailed' });
```

Comment:
do we need this? I assumed the error title set and error handling was done in the  `src/boot/init.ts`. Won't this be a duplicate code?

Reply:
init is separate